components/AppMenu.vue

```diff
@@ -13,6 +13,11 @@ function getFlagEmoji(countryCode: string) {
     .map((char) => 127397 + char.charCodeAt(0))
   return String.fromCodePoint(...codePoints)
 }
+
+async function changeLocale(locale: string) {
+  await loadLocaleMessages(locale)
```

Comment:
We have to load spanish translations file as we are using Lazy Loading strategy.
English and French are already loaded as English is the fallback and French the choosen one from the user.